mongo/integration/client_side_encryption_prose_test.go

```diff
@@ -1001,3 +1115,100 @@ func decodeJSONFile(mt *mtest.T, file string, val interface{}) bson.Raw {
 func rawValueToCoreValue(rv bson.RawValue) bsoncore.Value {
 	return bsoncore.Value{Type: rv.Type, Data: rv.Value}
 }
+
+type deadlockTest struct {
+	clientTest           *mongo.Client
+	clientKeyVaultOpts   *options.ClientOptions
+	clientKeyVaultEvents []startedEvent
+	clientEncryption     *mongo.ClientEncryption
+	ciphertext           primitive.Binary
+}
+type startedEvent struct {
```

Comment:
nit: We usually leave a blank line between type/function definitions unless there's a stylistic reason not to do so in this case.

Reply:
Done.